modules/image-stream-configure.adoc

```diff
@@ -0,0 +1,507 @@
+// Module included in the following assemblies:
+// * assembly/openshift_images
+
+[id='image-stream-configure_{context}']
+
+[[image-stream-configure]]
+= Configuring image streams
+
+ifdef::openshift-origin,openshift-online,openshift-enterprise,openshift-dedicated[]
+An image stream object file contains the following elements.
+endif::openshift-origin,openshift-online,openshift-enterprise,openshift-dedicated[]
+
+ifdef::openshift-origin,openshift-online,openshift-enterprise,openshift-dedicated[]
+[NOTE]
+====
+See the xref:../../dev_guide/managing_images.adoc#dev-guide-managing-images[Developer Guide] for details
```

Comment:
I would recommend removing all xrefs (content may not be available + we're not supposed to use them in the modular docs).

We can note the related content in the additional resources section or if appropriate, include the relevant module.